lib/dradis/plugins/csv/importer.rb

```diff
@@ -0,0 +1,19 @@
+module Dradis::Plugins::CSV
+  class Importer < Dradis::Plugins::Upload::Importer
+    def self.templates
+      {}
+    end
+
+    def import(params={})
+      logger.info { 'Parsing CSV file...' }
```

Comment:
Since we're not parsing the CSV here, can we change this to something like: `Uploading CSV file...`?